### PR TITLE
Add OAuth endpoint to credentials configuration.

### PR DESCRIPTION
**Pull request by eshtadc**

### CODE REVIEW

File: src/Service/Context/PageContext.php [hunk 1]
```diff
@@ -94,7 +95,9 @@ public function __construct(ConfigFactoryInterface $config_factory, EntityTypeMa
    */
   private function setPageContextCredential($credential_settings) {
     foreach (SELF::$CREDENTIAL_MAPPING as $credential_key => $tag_name) {
-      $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      if (isset($credential_settings[$credential_key]) {
```
Comment (ynx) [inline src/Service/Context/PageContext.php]: Should be 

```
if (isset($credential_settings[$credential_key])) {
```

File: src/Service/Context/PageContext.php [hunk 2]
```diff
@@ -94,7 +95,9 @@ public function __construct(ConfigFactoryInterface $config_factory, EntityTypeMa
...
+        $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      }
```
Comment (ynx) [inline src/Service/Context/PageContext.php:100]: I think we should always set the pageContext, but default to empty string. This way, a meta tag (although can be empty value) is always dropped on the page.

File: src/Service/Context/PageContext.php (hunk 1 above)
Comment (eshtadc) [inline src/Service/Context/PageContext.php]: D'OH.  fixing...

File: src/Service/Context/PageContext.php (hunk 2 above)
Comment (eshtadc) [inline src/Service/Context/PageContext.php:100]: Well - the JS code treats anything in the meta tags as what should override admin values.  I don't want an empty oauth url to override and I don't want to force someone to enter the auth url everytime.

Comment (ynx) [inline src/Service/Context/PageContext.php:100]: That's a good thought. 

On a semi-separated concern, does the JS mind if empty meta tags are simply skipped and not dropped on the page at all? If yes, should it?

**Comment by ynx** [comment]
https://github.com/acquia/acquia_lift/blob/63be493c15accd834c9041c657d0a061d16778e5/src/Tests/SettingsTest.php#L101

No longer 14, should be 15 since we have added a new field. All other unit / functional tests passing. After this fix we are ready to merge. Thank you @lkhaas!

File: src/Service/Context/PageContext.php (hunk 2 above)
Comment (eshtadc) [inline src/Service/Context/PageContext.php:100]: It doesn't mind if they are optional (like this one).  It has checks in place to ensure that credentials are available, etc. (regardless of how they are added to the page).

**Pull request closed without merging**